cnd/src/http_api/ethereum_network.rs

```diff
@@ -35,6 +37,25 @@ impl Network {
     }
 }
 
+impl From<ChainId> for Network {
```

Comment:
❓ Is this for backwards compatibility? Will it be removed in the future?

Reply:
Yes, it is used by the HTTP API, as per the issue's description.